requirements/portable-exe-extras.txt

```diff
@@ -1 +1,2 @@
 pyinstaller==5.12.0
+importlib_metadata>=7.0,<=7.1.0
```

Comment:
We should add a comment here why we are declaring `importlib_metadata` in this requirements file even though the build system does not directly require it.